src/PeoplePage.jsx

```diff
@@ -275,36 +278,81 @@ return (
             Followers
           </TabsButton>
         )}
+        <TabsButton
+          href={`${peopleUrl}?tab=trending`}
+          selected={state.selectedTab === "trending"}
+        >
+          Trending
+        </TabsButton>
+        {context.accountId && (
+          <TabsButton
+            href={`${peopleUrl}?tab=recommended`}
+            selected={state.selectedTab === "recommended"}
+          >
+            Recommended
+          </TabsButton>
+        )}
       </Tabs>
     )}
 
     {state.searchResults?.length === 0 && (
       <Text>No people matched your search.</Text>
     )}
 
-    {items.length > 0 && (
-      <Items>
-        {items.map((person, i) => (
-          <Item key={person.accountId}>
-            <Widget
-              src="${REPL_ACCOUNT}/widget/AccountProfileCard"
-              props={{
-                accountId: person.accountId,
-                blockHeight: person.blockHeight,
-              }}
-            />
-          </Item>
-        ))}
-      </Items>
+    {(state.selectedTab == "everyone" ||
+      state.selectedTab == "following" ||
+      state.selectedTab == "followers") &&
+      items.length > 0 && (
+        <Items>
+          {items.map((person, i) => (
+            <Item key={person.accountId}>
+              <Widget
+                src="${REPL_ACCOUNT}/widget/AccountProfileCard"
+                props={{
+                  accountId: person.accountId,
+                  blockHeight: person.blockHeight,
+                }}
+              />
+            </Item>
+          ))}
+        </Items>
+      )}
+
+    {!context.accountId && state.selectedTab == "trending" && (
+      <Widget
+        src="${REPL_ACCOUNT}/widget/Recommender.Views.TrendingUsersView"
+        props={{ currentPage: state.currentPage, fromContext: fromContext }}
+      />
+    )}
+
+    {context.accountId && state.selectedTab == "trending" && (
+      <Widget
+        src="${REPL_ACCOUNT}/widget/Recommender.Views.TrendingUsersView"
+        props={{ currentPage: state.currentPage, fromContext: fromContext }}
+      />
```

Comment:
Can't we just render it in this way:

```suggestion
    {state.selectedTab == "trending" && (
      <Widget
        src="${REPL_ACCOUNT}/widget/Recommender.Views.TrendingUsersView"
        props={{ currentPage: state.currentPage, fromContext: fromContext }}
      />
```
?

Reply:
One of the views is for logged out user experience. 
I think, it won't render if not mentioned explicitly